src/Microsoft.Azure.SignalR.Management/HubInstanceFactories/IServiceHubLifetimeManagerFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.SignalR.Management
+{
+    internal interface IServiceHubLifetimeManagerFactory
+    {
+        Task<IServiceHubLifetimeManager> Create(string hubName, CancellationToken cancellationToken);
```

Comment:
why both are required?

Reply:
Because passing a logger here is obsolete and I want to separate them.

Reply:
removed one.